feeds/cz.json

```diff
@@ -32,8 +36,17 @@
         },
         {
             "name": "IDS-JMK",
-            "type": "transitland-atlas",
-            "transitland-atlas-id": "f-u2e-idsjmk"
+            "type": "url",
+            "spec": "gtfs-rt",
+            "url": "https://content.idsjmk.cz/kestazeni/gtfsReal.dat"
+        },
+        {
+            "name": "IDS-JMK",
+            "type": "http",
+            "url": "https://www.arcgis.com/sharing/rest/content/items/379d2e9a7907460c8ca7fda1f3e84328/data"
```

Comment:
This is the same url that we were also getting from transitland. Using transitland is better, because we then don't need to deal with changes to the url.